### PR TITLE
Ref [TASK][38674] Navbar complete.

### PR DESCRIPTION
**Pull request by hi-long**
- Link redmine: https://edu-redmine.sun-asterisk.vn/issues/38674

### CODE REVIEW

**Comment by hi-long** [comment]
ready

File: src/pug/mixins/navbar/navLeft.pug [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+            each item in navLeftItems
+                +navLeftItem(item)
+
+//- NAV LEFT ITEM
+mixin navLeftItem(props)
+    if props.type == 'normal'
+        //- IF SCREEN < LG, "SAN PHAM" WITH NO DROPDOWN
+        if props.name == "SẢN PHẨM"
+            li.nav-item.p-1.fs-7.d-block.d-lg-none
+                a.nav-link.text-black(href=`${props.to}`)=    props.name
```
Comment (manhnt-0960) [inline src/pug/mixins/navbar/navLeft.pug]:
```suggestion
                a.nav-link.text-black(href=`${props.to}`)= props.name
```

Comment (manhnt-0960) [reply src/pug/mixins/navbar/navLeft.pug]: sau dấu = để 1 space thôi e

**Comment by hi-long** [comment]
ready

**Pull request merged**